controllers/remote/cluster_cache_reconciler.go

```diff
@@ -44,6 +45,7 @@ type ClusterCacheReconciler struct {
 
 func (r *ClusterCacheReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, options controller.Options) error {
 	err := ctrl.NewControllerManagedBy(mgr).
+		Named("remote/clustercache").
```

Comment:
@fabriziopandini I set the name here.

This will lead to a "controller": "remote/clustercache" k/v pair.

Before this PR the logs from this controller had "controller": "cluster" (same as the cluster controller).


Note: This info is based on controller-runtime main. The current v0.11 release will only use the name as prefix for messages.